config/services.yaml

```diff
@@ -0,0 +1,42 @@
+auth-service:
```

Comment:
This file should be instead `services.yaml.example` and the creation of `services.yaml.example` should be included on readme file. This is important since we will change this file frequently and if versioned, whenever there's a change, there will be a git conflict